src/containers/Rubric/Rubric.scss

```
@@ -49,15 +48,15 @@
   width: 320px !important;
   height: fit-content;
   max-height: 100%;
-  margin-left: map-get($spacers, 3);
+  margin-left: var(--pgn-spacing-spacer-3);
   position: sticky !important;
-  top: map-get($spacers, 1) * -1;
+  top: calc(calc(var(--pgn-spacing-spacer-1) * -1));
```

[nit]: It seems we have an extra calc() here.
For example: calc(2 * -1) = -2

```suggestion
  top: calc(var(--pgn-spacing-spacer-1) * -1);
```

---

Fixed